py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -276,30 +277,48 @@ def _populate_trt_builder_config(
     def _create_timing_cache(
         self,
         builder_config: trt.IBuilderConfig,
-        existing_cache: Optional[trt.ITimingCache] = None,
-    ) -> trt.ITimingCache:
-        cache = None
-        if existing_cache:
-            cache_file = np.array(existing_cache)
-            cache = builder_config.create_timing_cache(cache_file.tobytes())
-        else:
-            cache = builder_config.create_timing_cache(b"")
+        timing_cache_path: str = "",
+    ) -> None:
+        """
+        Create a timing cache to enable faster build time for TRT engines.
+        By default the timing_cache_path="/tmp/timing_cache.bin"
+        """
+        buffer = b""
+        if os.path.exists(timing_cache_path):
+            # Load from existing cache
+            with open(timing_cache_path, mode="rb") as timing_cache_file:
+                buffer = timing_cache_file.read()
+        cache = builder_config.create_timing_cache(buffer)
         builder_config.set_timing_cache(cache, False)
-        return cache
+
+    def _save_timing_cache(
+        self,
+        builder_config: trt.IBuilderConfig,
+        timing_cache_path: str = "",
+    ) -> None:
+        """
+        This is called after a TensorRT engine is built. Save the timing cache
+        """
+        timing_cache = builder_config.get_timing_cache()
+        if timing_cache_path:
+            with open(timing_cache_path, "wb") as timing_cache_file:
+                timing_cache_file.write(memoryview(timing_cache.serialize()))
+        else:
+            tmp_dir_path = os.path.join(tempfile.gettempdir(), "timing_cache.bin")
```

Comment:
Is it not preferred that simply assign `TIMING_CACHE_PATH` in `_defaults.py` to `os.path.join(tempfile.gettempdir(), "timing_cache.bin")`? Otherwise `_create_timing_cache` will always receive an empty string by default and hence a new cache will be created every time.

Reply:
I don't have a strong preference here. With the first run, there's a possibility that a torch-trt session might be using a timing cache from another model, but there's no harm in that I guess.